docs/versioned_docs/version-2.11.0/user-guides/prerequisites.mdx

```diff
@@ -0,0 +1,500 @@
+---
+id: prerequisites
+title: OpenEBS Install Prerequisites
+---
+
+import ImgCard from "@site/src/components/ImgCard";
+
+This guide will help you to verify that your Kubernetes worker nodes have the required prerequisites to install OpenEBS and use OpenEBS Volumes to run your Kubernetes Stateful Workloads. In addition, you will learn about how to customize the installer according to your managed Kubernetes provider.
+
+OpenEBS provides block volume support through the iSCSI protocol. Therefore, 
+the iSCSI client (initiator) presence on all Kubernetes nodes is required.
+Choose the platform below to find the steps to verify if the iSCSI client
+is installed and running or to find the steps to install the iSCSI client.
+
+#### Choose the platform for iSCSI client settings
+
+<ImgCard
+  dataList={[
+    {
+      Svg: "/docs/img/logos/l-ubuntu.png",
+      title: "Ubuntu",
+      url: "/user-guides/prerequisites#ubuntu",
+    },
+    {
+      Svg: "/docs/img/logos/l-rhel.png",
+      title: "RHEL",
+      url: "/user-guides/prerequisites#rhel",
+    },
+    {
+      Svg: "/docs/img/logos/l-centos.png",
+      title: "CentOS",
+      url: "/user-guides/prerequisites#centos",
+    },
+    {
+      Svg: "/docs/img/logos/l-eks.png",
+      title: "EKS",
+      url: "/user-guides/prerequisites#eks",
+    },
+    {
+      Svg: "/docs/img/logos/l-gke.png",
+      title: "GKE",
+      url: "/user-guides/prerequisites#gke",
+    },
+    {
+      Svg: "/docs/img/logos/l-aks.png",
+      title: "AKS",
+      url: "/user-guides/prerequisites#aks",
+    },
+    {
+      Svg: "/docs/img/logos/l-openshift.png",
+      title: "OpenShift",
+      url: "/user-guides/prerequisites#openshift",
+    },
+    {
+      Svg: "/docs/img/logos/l-rancher.png",
+      title: "Rancher",
+      url: "/user-guides/prerequisites#rancher",
+    },
+    {
+      Svg: "/docs/img/logos/l-icp.png",
+      title: "ICP",
+      url: "/user-guides/prerequisites#icp",
+    },
+    {
+      Svg: "/docs/img/logos/DigitalOceanLogo.png",
+      title: "DigitalOcean",
+      url: "/user-guides/prerequisites#do",
+    },
+    {
+      Svg: "/docs/img/logos/d2iq.jpg",
+      title: "Konvoy",
+      url: "/user-guides/prerequisites#konvoy",
+    },
+    
+  ]}
+/>
+
+<br/>
```

Comment:
please remove `<br />` tag

Reply:
The `<br/>` given intentionally, there was no space coming after the `ImgCard`